contracts/test/v0.8/automation/KeeperRegistry1_2.test.ts

```diff
@@ -17,6 +17,27 @@ import { UpkeepMock } from '../../../typechain/UpkeepMock'
 import { UpkeepTranscoder } from '../../../typechain/UpkeepTranscoder'
 import { toWei } from '../../test-helpers/helpers'
 
+//////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////////////////////
+
+/*********************************** REGISTRY v1.2 IS FROZEN ************************************/
+
+// All tests are disabled for this contract, as we expect it to never change in the future.
+// Instead, we test that the bytecode for the contract has not changed.
```

Comment:
Can we expand on the comment here?

"If this test fails and you have to change the bytecode, make sure to unfreeze the main test suite"

Reply:
done :+1: